spearhead/assets/js/extend-blocks.js

```diff
@@ -0,0 +1,20 @@
+/* global wp */
+( function( blocks, domReady ) {
+    var styles = [
+        {
+            name: "core/group",
```

Comment:
What do we need this for?

Reply:
I left this in here as a pattern to follow for adding block styles. It's unused at the moment.